workflows/codefresh-csdp/versions/0.0.1/workflowTemplate.yaml

```diff
@@ -74,3 +74,62 @@ spec:
             value: '{{ inputs.parameters.GIT_PROVIDER_NAME }}'
           - name: CF_API_KEY
             value: '{{ inputs.parameters.CF_API_KEY }}'
+    - name: image-enricher-jira
+      serviceAccountName: argo-hub.codefresh-csdp.0.0.1
+      metadata:
+        annotations:
+          argo-hub-template/description: 'Enrich codefresh image with jira info'
+          argo-hub-template/icon_url: "https://cdn.jsdelivr.net/gh/codefresh-io/argo-hub@main/workflows/codefresh-csdp/assets/icon.svg"
+          argo-hub-template/icon_background: "#171A2D"
+      retryStrategy:
+        limit: "10"
+        retryPolicy: "Always"
+        backoff:
+          duration: "5s"
+      inputs:
+        parameters:
+          - name: CF_URL
+            default: ''
+          - name: FAIL_ON_NOT_FOUND
+            default: 'false'
+          - name: JIRA_CONTEXT
+            default: ''
+          - name: JIRA_EMAIL
+            default: ''
+          - name: JIRA_HOST
+            default: ''
+          - name:  JIRA_API_TOKEN
+            default: ''
+          - name: JIRA_PROJECT_PREFIX
+          - name: CF_API_KEY
+          - name: IMAGE
+          - name: MESSAGE
+
+      container:
+        name: main
+        imagePullPolicy: Always
+        image: quay.io/codefreshplugins/argo-hub-workflows-codefresh-csdp-versions-0.0.1-images-image-enricher-jira:main
+        command:
+          - node
+          - /app/src/index.js
+        env:
+          - name: CF_URL
+            value: '{{ inputs.parameters.CF_URL }}'
+          - name: FAIL_ON_NOT_FOUND
+            value: '{{ inputs.parameters.FAIL_ON_NOT_FOUND }}'
+          - name: JIRA_CONTEXT
+            value: '{{ inputs.parameters.JIRA_CONTEXT }}'
+          - name: JIRA_EMAIL
+            value: '{{ inputs.parameters.JIRA_EMAIL }}'
+          - name: JIRA_HOST
+            value: '{{ inputs.parameters.JIRA_HOST }}'
+          - name: JIRA_API_TOKEN
+            value: '{{ inputs.parameters.JIRA_API_TOKEN }}'
+          - name: JIRA_PROJECT_PREFIX
+            value: '{{ inputs.parameters.JIRA_PROJECT_PREFIX }}'
+          - name: CF_API_KEY
```

Comment:
this is not a correct way to pass secrets.
it should be read from k8s secret. 
check how i implemented the report commit status github